src/common/datomish/db.cljc

```diff
@@ -383,7 +379,7 @@
             (ref? a)           ; index_vaet
             (unique? a)        ; unique_value
             tag]]))
-      ops
+      (sort-by (fn [[_ _ _ v]] v) ops) ;; Make testing easier by sorting by string values.  TODO: discuss expense.
```

Comment:
We could, I suppose, by I really don't expect an in-memory sort -- even of fulltext values -- to be costly.  Let's wait and see if it shows up when we profile, and whether there's _another_ situation that such a flag would be useful.